test/test_modules.py

```diff
@@ -165,10 +166,9 @@ def test_convnet(
     )
     if in_features is None:
         in_features = 5
-    x = torch.randn(batch, in_features, input_size, input_size, device=device)
+    x = torch.randn(batch, seq_length, in_features, input_size, input_size, device=device)
```

Comment:
Here's how i would do it
we can add a 
```python
@pytest.mark.parametrize("batch", [tuple(), (2,), (2, 3, 2)])
def test_convnet(..., batch):
    ...
    x = torch.randn(*batch, in_features, input_size, input_size, device=device)
```

like that we test (no batch - batch of dim 1 - batch of dim > 1)